src/components/ProductTours/data/hooks.js

```diff
@@ -14,8 +13,9 @@ import { EnterpriseSubsidiesContext } from '../../EnterpriseSubsidiesContext';
 const cookies = new Cookies();
 
 export const useBrowseAndRequestTour = () => {
-  const history = useHistory();
-  const inSettingsPage = history.location.pathname.includes(ROUTE_NAMES.settings);
+  const { params: { page } } = useRouteMatch();
```

Comment:
nit: could use `useParams` here if we're only using the params from `useRouteMatch`